index.html

```diff
@@ -4674,23 +4674,139 @@ <h2>
-specifications will recognize control from <a>DIDs</a> registered using other
-<a>DID methods</a> and they might restrict third-party control to <a>DIDs</a>
-that use the same method. Access control and key recovery in a
-<a>DID method</a> specification can also include a time lock feature to
-protect against key compromise by maintaining a second track of control for
-recovery.
-</p>
+        Section <a href="#method-operations"></a> specifies the
+        <a>DID</a> operations to be supported by a
+        <a>DID method</a> specification, including
+        <a href="#update">update</a>
+        which MUST be used to perform a verification method rotation.
+      </p>
+  
+      <p>
+        A controller performs a rotation when they add a new verification
+        method that is meant to replace an existing verification method after
+        some time.
+      </p>
+  
+      <p>Not all DID Methods support verification method rotation.</p>
+  
+      <p>
+        Unlike revocation, rotation does not cause existing proofs intended
+        to be verified by the old verification method to be considered invalid. Rather,
+        rotation is a key management process that enables the private cryptographic
+        material associated with an existing verification method to be deactivated
+        or destroyed once the new verification method has been added to the DID
+        Document. Going forward, any new proofs that a controller would have generated
+        using the old cryptographic material can now instead be generated using the new
+        material and can be verified using the new verification method.
+      </p>
+  
+      <p>
+        Rotation is a useful mechanism for protecting against verification
+        method compromise, since frequent rotation of a verification method by
+        the controller reduces the value of a single compromised verification method
+        to an attacker. Performing revocation immediately after rotation is
+        useful for verification methods that a controller designates for
+        short-lived verifications, such as those involved in encrypting messages
+        and authentication.
+      </p>
+  
+      <p class="note">
+        Higher security environments tend to employ more frequent verification
+        method rotation.
+      </p>
+  
+      <p class="note">
+        Frequent rotation of a verification method might be frustrating for
+        parties that are forced to continiously renew or refresh associated
+        credentials.
+      </p>
+    </section>
+  
+    
+  
+    <section>
+      <h2>DID Recovery</h2>
+  
+      <p>
+        Recovery is a reactive security measure, whereby a controller is able
+        to regain the ability to perform DID operations.
+      </p>
+  
+      <p class="advisement">
+        Recovery is advised when a controller or services trusted to act on
+        their behalf no longer has the exclusive ability to perform DID
```

Comment:
`have` just doesn't feel right here. Conflicting singular and plural subjects make it hard to choose between singular and plural verb forms.

Maybe this, which is purely singular?
```suggestion
        Recovery is advised when a controller or service trusted to act on
        their behalf no longer has the exclusive ability to perform DID
```

Reply:
I'm fine with the change going either way.

Reply:
I am resolving this as I cannot apply it, because it is outdated.